apps/system/lockscreen/js/state/lockscreen_basic_state.js

```diff
@@ -57,6 +61,17 @@
    * Or the component cannot track the last active state.
    */
   LockScreenBasicState.prototype.transferTo = function() {
+    if (this._transferred) {
```

Comment:
This is a simple deduction progress:

1. Just focus on this method: whether calling a State to transfer itself twice is reasonable or not. 
2. Obviously it's not reasonable. So, how to make sure this wouldn't happen?
3.1. We could ask every caller to detect whether the state is transferred.
3.2. Or the method could 'nullify' it's second call
4. However, to adopt 3.1 would, as I mentioned, to ask every state to do the duplicated thing to detect the similar condition using the similar code
5. So here I adopted 3.2, to solve this case while prevent from scattering duplicated code about everywhere